### PR TITLE
if sort and accessor defined, only use sort for sorting

### PR DESCRIPTION
**Pull request by joefitter**

### CODE REVIEW

File: pages/project/list/schema/index.js [hunk 1]
```diff
@@ -5,6 +5,7 @@ module.exports = {
   },
   licenceHolder: {
     show: true,
+    sort: 'licenceHolder.lastName',
     accessor: 'licenceHolder.name',
```
Comment (lennym) [inline pages/project/list/schema/index.js]: We might want to get rid of this anyway, and use a formatter to render the full name from the model instead on account of the fact that the `name` virtual property is flaky, and should be considered deprecated.

Comment (joefitter) [reply pages/project/list/schema/index.js]: yes agree, however the `sort` property is needed as we need to sort on lastName

Comment (joefitter) [reply pages/project/list/schema/index.js]: that was my way of saying "I'm staying in my lane", but actually I'm gunna swerve into oncoming traffic and fix this. Please hold.

Comment (lennym) [reply pages/project/list/schema/index.js]: Shall I merge then?

File: pages/project/list/views/index.jsx [hunk 2]
```diff
@@ -12,7 +12,7 @@ import {
 
 export const formatters = {
   licenceHolder: {
-    format: (name, project) => <Link page="profile.view" profile={project.licenceHolder.id} label={ name } />
+    format: ({ id, firstName, lastName }) => <Link page="profile.view" profile={id} label={`${firstName} ${lastName}`} />
```
Comment (lennym) [inline pages/project/list/views/index.jsx]: 👍

**Pull request merged**